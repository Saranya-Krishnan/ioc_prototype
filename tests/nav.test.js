import React from 'react';
import Nav from '../views/src/components/nav';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router'
import { Link } from 'react-router-dom';

test('it displays the correct navigation items', () =>{
    const clickTest = () => {
      return true;
    };
    const component = renderer.create(
        <MemoryRouter>
            <Nav activeItem={'test'} clickMenuItem={clickTest}></Nav>
        </MemoryRouter>
    );
    let c = component.toJSON();
    expect(c).toMatchSnapshot();
});