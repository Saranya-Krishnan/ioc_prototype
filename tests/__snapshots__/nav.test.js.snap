// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it displays the correct navigation items 1`] = `
<div
  className="ui text container"
>
  <div
    className="ui pointing secondary menu"
  >
    <a
      className="item"
      href="/"
      onClick={[Function]}
    >
      home
    </a>
    <div
      className="right menu"
    >
      <a
        className="item"
        href="/sign-in"
        onClick={[Function]}
      >
        sign in
      </a>
      <a
        className="item"
        href="/sign-up"
        onClick={[Function]}
      >
        sign up
      </a>
    </div>
  </div>
</div>
`;
